tools/clang/unittests/HLSL/ValidationTest.cpp

```diff
@@ -1057,6 +1059,8 @@ TEST_F(ValidationTest, SimpleHs1Fail) {
       });
 }
 TEST_F(ValidationTest, SimpleHs3Fail) {
+  if (m_ver.SkipDxilVersion(1, 8))
+    return;
```

Comment:
SimpleHs1Fail was the test that was altered in #6196. I think this one is fine and that one needs this protection.

Reply:
:( Fixed.